x-pack/docs/en/rest-api/graph/explore.asciidoc

```diff
@@ -56,7 +56,7 @@ Elasticsearch query. For example:
   }
 }
 --------------------------------------------------
-
+// NOTCONSOLE
```

Comment:
I think we'd be much better off writing a real request with just the `query` part in it or something like that. We had a lot of "unrooted" JSON in our docs before we had the docs tests and removing it all made the docs a ton more clear. No more wondering "where does this go"? Also, the way this is it doesn't get any test coverage.

Reply:
I recall the response parts were tricky to test given the scores produced and how they might change with Lucene versions etc. and the difficulty involved in masking them from any "actual vs expected" result comparisons.

Reply:
It is possible to ignore the numbers with tricks like `// TESTRESPONSE[s/12.3/$body.$_path/]`.